Readme.md

```diff
@@ -44,7 +44,7 @@ Legacy *synchronous* client that supports:
 [![Build Status](https://github.com/nats-io/nats.rs/workflows/Rust/badge.svg)](https://github.com/nats-io/nats.rs/actions)
 
 
-New bleeding edge experimental async Tokio-based NATS client.
+New async Tokio-based NATS client.
```

Comment:
Still want us to be explicit about us being pre 1.x and API changes may occur from time to time.